app/src/test/java/com/bitwarden/authenticator/data/authenticator/manager/util/VerificationCodeItemUtil.kt

```diff
@@ -15,12 +16,14 @@ fun createMockVerificationCodeItem(
 ): VerificationCodeItem =
     VerificationCodeItem(
         code = "mockCode-$number",
-        totpCode = "mockTotpCode-$number",
         periodSeconds = 30,
         timeLeftSeconds = 120,
         issueTime = 0,
         id = "mockId-$number",
-        username = "mockUsername-$number",
+        accountName = "mockLabel-$number",
```

Comment:
⛏️ Should be `mockAccountName`, but this is a test helper so up to you if you want to update it. As long as the tests pass, :shipit:.